### PR TITLE
Jannes changes

### PR DESCRIPTION
**Pull request by jannes-m**
addresses #996. I think the changes are all pretty non-controversial.

### CODE REVIEW

File: 04-spatial-operations.Rmd [hunk 1]
```diff
@@ -1115,7 +1115,7 @@ Aside from that, global operations are also useful for the computation of distan
 In the first case, one can calculate the distance from each cell to a specific target cell.
 For example, one might want to compute the distance to the nearest coast (see also `terra::distance()`).
 We might also want to consider topography, that means, we are not only interested in the pure distance but would like also to avoid the crossing of mountain ranges when going to the coast.
-To do so, we can weight the distance with elevation so that each additional altitudinal meter 'prolongs' the Euclidean distance.
+To do so, we can weight the distance with elevation so that each additional altitudinal meter 'prolongs' the Euclidean distance (in Exercises 8 and 9 at the end of this chapter you will do exactly that).
```
Comment (Robinlovelace) [inline 04-spatial-operations.Rmd:1118]: :+1:

File: 12-spatial-cv.Rmd [hunk 2]
```diff
@@ -21,15 +21,15 @@ Packages **GGally**, **lgr**, **kernlab**, **mlr3measures**, **paradox**, **pROC
...
-library(future)
-library(lgr)
-library(mlr3)
-library(mlr3learners)
-library(mlr3extralearners)
-library(mlr3spatiotempcv)
-library(mlr3tuning)
-library(mlr3viz)
-library(progressr)
+library(future)             # parallel processing
```
Comment (Robinlovelace) [inline 12-spatial-cv.Rmd:24]: :+1:

**Review by Robinlovelace — APPROVED** [review]

File: 15-eco.Rmd [hunk 3]
```diff
@@ -40,27 +40,25 @@ Every few years, the El Niño phenomenon brings torrential rainfall to this sun-
...
+Specifically, we will:
 
-Ordinations\index{ordination} are dimension-reducing techniques that allow the extraction of the main gradients from a (noisy) dataset, in our case the floristic gradient developing along the southern mountain slope (see next section).
-In this chapter, we will model the first ordination axis, i.e., the floristic gradient, as a function of environmental predictors such as altitude, slope, catchment area\index{catchment area} and NDVI\index{NDVI}.
-For this, we will make use of a random forest model\index{random forest} --- a very popular machine learning\index{machine learning} algorithm [@breiman_random_2001].
-The model will allow us to make spatial distribution maps of the floristic composition anywhere in the study area.
-To guarantee an optimal prediction, it is advisable to tune beforehand the hyperparameters\index{hyperparameter} with the help of spatial cross-validation\index{cross-validation!spatial CV} (see Section \@ref(svm)).
+- load in needed data and compute environmental predictors (Section \@ref(data-and-data-preparation)).
```
Comment (Nowosad) [inline 15-eco.Rmd:57]:
```suggestion
- Load in needed data and compute environmental predictors (Section \@ref(data-and-data-preparation))
```

File: 15-eco.Rmd [hunk 4]
```diff
@@ -40,27 +40,25 @@ Every few years, the El Niño phenomenon brings torrential rainfall to this sun-
...
+- extract the main floristic gradient from our species composition matrix with the help of a dimension-reducing technique (ordinations\index{ordination}; Section @ref(nmds)).
```
Comment (Nowosad) [inline 15-eco.Rmd:58]:
```suggestion
- Extract the main floristic gradient from our species composition matrix with the help of a dimension-reducing technique (ordinations\index{ordination}; Section @ref(nmds))
```

File: 15-eco.Rmd [hunk 5]
```diff
@@ -40,27 +40,25 @@ Every few years, the El Niño phenomenon brings torrential rainfall to this sun-
...
+- model the first ordination axis, i.e., the floristic gradient, as a function of environmental predictors such as altitude, slope, catchment area\index{catchment area} and NDVI\index{NDVI} (Section \@ref(modeling-the-floristic-gradient)).
```
Comment (Nowosad) [inline 15-eco.Rmd:59]:
```suggestion
- Model the first ordination axis, i.e., the floristic gradient, as a function of environmental predictors such as altitude, slope, catchment area\index{catchment area} and NDVI\index{NDVI} (Section \@ref(modeling-the-floristic-gradient))
```

File: 15-eco.Rmd [hunk 6]
```diff
@@ -40,27 +40,25 @@ Every few years, the El Niño phenomenon brings torrential rainfall to this sun-
...
+For this, we will make use of a random forest model\index{random forest} --- a very popular machine learning\index{machine learning} algorithm [@breiman_random_2001]. To guarantee an optimal prediction, it is advisable to tune beforehand the hyperparameters\index{hyperparameter} with the help of spatial cross-validation\index{cross-validation!spatial CV} (see Section \@ref(svm)).
+- make a spatial distribution map of the floristic composition anywhere in the study area (Section \@ref(predictive-mapping)).
```
Comment (Nowosad) [inline 15-eco.Rmd:61]:
```suggestion
- Make a spatial distribution map of the floristic composition anywhere in the study area (Section \@ref(predictive-mapping))
```

File: 15-eco.Rmd [hunk 7]
```diff
@@ -40,27 +40,25 @@ Every few years, the El Niño phenomenon brings torrential rainfall to this sun-
...
-To guarantee an optimal prediction, it is advisable to tune beforehand the hyperparameters\index{hyperparameter} with the help of spatial cross-validation\index{cross-validation!spatial CV} (see Section \@ref(svm)).
+- load in needed data and compute environmental predictors (Section \@ref(data-and-data-preparation)).
+- extract the main floristic gradient from our species composition matrix with the help of a dimension-reducing technique (ordinations\index{ordination}; Section @ref(nmds)).
+- model the first ordination axis, i.e., the floristic gradient, as a function of environmental predictors such as altitude, slope, catchment area\index{catchment area} and NDVI\index{NDVI} (Section \@ref(modeling-the-floristic-gradient)).
+For this, we will make use of a random forest model\index{random forest} --- a very popular machine learning\index{machine learning} algorithm [@breiman_random_2001]. To guarantee an optimal prediction, it is advisable to tune beforehand the hyperparameters\index{hyperparameter} with the help of spatial cross-validation\index{cross-validation!spatial CV} (see Section \@ref(svm)).
```
Comment (Nowosad) [inline 15-eco.Rmd:60]:
```suggestion
For this, we will make use of a random forest model\index{random forest} --- a very popular machine learning\index{machine learning} algorithm [@breiman_random_2001]. To guarantee an optimal prediction, it is advisable to tune beforehand the hyperparameters\index{hyperparameter} with the help of spatial cross-validation\index{cross-validation!spatial CV} (see Section \@ref(svm))
```

**Review by Nowosad — APPROVED** [review]

**Comment by Nowosad** [comment]
👍🏻

**Pull request merged**